pkg/gui/keybindings.go

```diff
@@ -66,108 +263,194 @@ func (b *Binding) GetKey() string {
 	return string(key)
 }
 
+var keymap = map[string]interface{}{
+	"<c-a>":       gocui.KeyCtrlA,
+	"<c-b>":       gocui.KeyCtrlB,
+	"<c-c>":       gocui.KeyCtrlC,
+	"<c-d>":       gocui.KeyCtrlD,
+	"<c-e>":       gocui.KeyCtrlE,
+	"<c-f>":       gocui.KeyCtrlF,
+	"<c-g>":       gocui.KeyCtrlG,
+	"<c-h>":       gocui.KeyCtrlH,
+	"<c-i>":       gocui.KeyCtrlI,
+	"<c-j>":       gocui.KeyCtrlJ,
+	"<c-k>":       gocui.KeyCtrlK,
+	"<c-l>":       gocui.KeyCtrlL,
+	"<c-m>":       gocui.KeyCtrlM,
+	"<c-n>":       gocui.KeyCtrlN,
+	"<c-o>":       gocui.KeyCtrlO,
+	"<c-p>":       gocui.KeyCtrlP,
+	"<c-q>":       gocui.KeyCtrlQ,
+	"<c-r>":       gocui.KeyCtrlR,
+	"<c-s>":       gocui.KeyCtrlS,
+	"<c-t>":       gocui.KeyCtrlT,
+	"<c-u>":       gocui.KeyCtrlU,
+	"<c-v>":       gocui.KeyCtrlV,
+	"<c-w>":       gocui.KeyCtrlW,
+	"<c-x>":       gocui.KeyCtrlX,
+	"<c-y>":       gocui.KeyCtrlY,
+	"<c-z>":       gocui.KeyCtrlZ,
+	"<c-~>":       gocui.KeyCtrlTilde,
+	"<c-2>":       gocui.KeyCtrl2,
+	"<c-3>":       gocui.KeyCtrl3,
+	"<c-4>":       gocui.KeyCtrl4,
+	"<c-5>":       gocui.KeyCtrl5,
+	"<c-6>":       gocui.KeyCtrl6,
+	"<c-7>":       gocui.KeyCtrl7,
+	"<c-8>":       gocui.KeyCtrl8,
+	"<c-space>":   gocui.KeyCtrlSpace,
+	"<c-\\>":      gocui.KeyCtrlBackslash,
+	"<c-[>":       gocui.KeyCtrlLsqBracket,
+	"<c-]>":       gocui.KeyCtrlRsqBracket,
+	"<c-/>":       gocui.KeyCtrlSlash,
+	"<c-_>":       gocui.KeyCtrlUnderscore,
+	"<backspace>": gocui.KeyBackspace,
+	"<tab>":       gocui.KeyTab,
+	"<enter>":     gocui.KeyEnter,
+	"<esc>":       gocui.KeyEsc,
+	"<space>":     gocui.KeySpace,
+	"<f1>":        gocui.KeyF1,
+	"<f2>":        gocui.KeyF2,
+	"<f3>":        gocui.KeyF3,
+	"<f4>":        gocui.KeyF4,
+	"<f5>":        gocui.KeyF5,
+	"<f6>":        gocui.KeyF6,
+	"<f7>":        gocui.KeyF7,
+	"<f8>":        gocui.KeyF8,
+	"<f9>":        gocui.KeyF9,
+	"<f10>":       gocui.KeyF10,
+	"<f11>":       gocui.KeyF11,
+	"<f12>":       gocui.KeyF12,
+	"<insert>":    gocui.KeyInsert,
+	"<delete>":    gocui.KeyDelete,
+	"<home>":      gocui.KeyHome,
+	"<end>":       gocui.KeyEnd,
+	"<pgup>":      gocui.KeyPgup,
+	"<pgdown>":    gocui.KeyPgdn,
+	"<up>":        gocui.KeyArrowUp,
+	"<down>":      gocui.KeyArrowDown,
+	"<left>":      gocui.KeyArrowLeft,
+	"<right>":     gocui.KeyArrowRight,
+}
+
+func (gui *Gui) getKey(name string) interface{} {
+	key := gui.Config.GetUserConfig().GetString("keybinding." + name)
+	if len(key) > 1 {
+		binding := keymap[strings.ToLower(key)]
+		if binding == nil {
+			log.Fatal("Unrecognized binding: " + strings.ToLower(key))
```

Comment:
This could be a little more descriptive
```suggestion
			log.Fatalf("Unrecognized key %s for keybinding %s", strings.ToLower(key), name)
```